### PR TITLE
Streaming errors

### PR DESCRIPTION
**Pull request by Dspil**
Instead of printing the  verification errors after the verification for each package ends, stream them.

### CODE REVIEW

File: src/main/scala/viper/gobra/backend/BackendVerifier.scala
Comment (ArquintL) [inline src/main/scala/viper/gobra/backend/BackendVerifier.scala:52]: @Dspil Would it be possible to set the `StreamingReporter` in the config instead of wrapping the reporter here?
What I image is that the reporter (or default reporter) for Gobra via CLI is the streaming reporter (maybe streaming reporter wrapping the current reporter). In this way, Gobra-IDE could use the existing reporter (i.e. a different one) without observing any changes

File: src/main/scala/viper/gobra/reporting/StreamingReporter.scala
Comment (jcp19) [inline src/main/scala/viper/gobra/reporting/StreamingReporter.scala:17]: This way, we will only log the errors if we are using the `StreamingReporter`, which will probably not be the case with the IDE right? Could we do the reporting somewhere else, like in `BacktranslatingReporter`?

Comment (ArquintL) [reply src/main/scala/viper/gobra/reporting/StreamingReporter.scala:17]: Although this is correct, I don't think errors printed to StdOut should be of concern for the IDE

File: src/main/scala/viper/gobra/reporting/StreamingReporter.scala
Comment (jcp19) [inline src/main/scala/viper/gobra/reporting/StreamingReporter.scala:20]:
```suggestion
        case _ =>
      }
      case _ =>
```

**Review by jcp19 — APPROVED** [review]
LGTM but I would wait for Linard's approval to confirm that this has no impact in the GobraIDE

**Pull request merged**
